src/middlewared/middlewared/alembic/versions/12.0/2020-05-18_14-29_nmbclusters-fix.py

```diff
@@ -0,0 +1,98 @@
+"""remove nmbclusters sysctl
+
+Revision ID: 730c995cbd37
+Revises: 434ea5397cd3
+Create Date: 2020-05-18 14:29:59.887895+00:00
+
+"""
+from alembic import op
+import subprocess as subp
+from middlewared.utils import osc
+
+
+# revision identifiers, used by Alembic.
+revision = '730c995cbd37'
+down_revision = '434ea5397cd3'
+branch_labels = None
+depends_on = None
+
+
+def get_db_sysctl_value():
+    TABLE = 'system_tunable'
+    SELECT_FROM = f'SELECT * FROM {TABLE}'
+    WHERE = ' WHERE tun_var = "kern.ipc.nmbclusters"'
```

Comment:
Should we also filter for comment = Generated by autotune?

Reply:
Yeah, I thought about that too. I'll filter for that as well.